tools/configure_server_settings.rb

```diff
@@ -32,7 +32,7 @@
 keys.each { |p| path = path[p.to_sym] }
 
 puts "Setting [#{opts[:path]}], old value: [#{path[key]}], new value: [#{opts[:value]}]"
-path[key] = opts[:value]
+path[key] = eval(opts[:value])
```

Comment:
Would it be better to create a new option for true / false?  Isn't this going to blow up for string values?